docs/src/tutorial/appendix.rst

```diff
@@ -14,7 +14,7 @@ Appendix: Installing MinGW on Windows
     includes e.g. "c:\\mingw\\bin" (if you installed MinGW to
     "c:\\mingw"). The following web-page describes the procedure
     in Windows XP (the Vista procedure is similar):
-    http://support.microsoft.com/kb/310519
+    https://support.microsoft.com/en-us/help/310519/how-to-manage-environment-variables-in-windows-xp
```

Comment:
Switching to https is fine, but the redirect is ok here since it keeps the link free of redundancy and potentially non-static parts.